pkg/driverbuilder/local.go

```diff
@@ -67,24 +67,35 @@ func (lbp *LocalBuildProcessor) Start(b *builder.Build) error {
 		// if an unsupported target is passed.
 		// Go on skipping automatic kernel headers download.
 		if err == nil {
-			slog.Info("Trying automatic kernel headers download.")
+			slog.Info("Trying automatic kernel headers download")
```

Comment:
Dropped some `.` at the end of log strings for `local` executor, for consistency.